lib/OpenQA/Utils.pm

```diff
@@ -224,6 +224,27 @@ sub needle_info {
     $needle->{name}      = $name;
     $needle->{distri}    = $distri;
     $needle->{version}   = $version;
+
+	#After change the needle's properties form to hash, support backward compatibility
```

Comment:
I'm still in favor of having a space after the `#`.

Reply:
I will fix it.